client/README.md

```diff
@@ -66,9 +66,8 @@ const (
 
 func main() {
 	// Make client
-	u, _ := url.Parse("http://localhost:8086")
-	c := client.NewClient(client.Config{
-		URL: u,
+	c := client.NewHTTPClient(client.HTTPConfig{
```

Comment:
Aw, sorry, if so, the README looks little bit confusing – I've used `import "github.com/influxdb/influxdb/client/v2"` with old API (`NewClient`) and now just updated code via `go get github.com/influxdb/influxdb` and have find that my code no longer works. If `v2` is like `next` or something, may be it should not have such name, I think... :confused: